### PR TITLE
release/1.3.0: Documentation update for most site configs

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

Documentation updates only:
- most site configs updated by now (the missing ones may or may not be done before we cut the release - adjust in final PR as needed)
- Complete the overview page

Rendered documentation can be seen here: https://spack-stack--526.org.readthedocs.build/en/526/

### CODE REVIEW

**Comment by climbfuji** [comment]
@Hang-Lei-NOAA @ulmononian What are the chances that the missing sites (Parallel Works, Acorn) will be done in the next days?

I will need to go back and look at Gaea, whether the problems with running single-task MPI jobs still occur.

Updates for those three sites can be made in a follow-up PR.

**Comment by mark-a-potts** [comment]
> 

I finished installing on all three ParallelWorks sites yesterday. The installs required statically built versions of MAPL and ESMF, but work with the cpld_control_p8 regression test of the WM. I am pretty sure we will need the statically built ESMF/MAPL combination on all the platforms to get the WM to work correctly.

File: doc/source/MaintainersSection.rst [hunk 1]
```diff
@@ -370,7 +370,7 @@ NCAR-Wyoming Cheyenne
...
 
 On Cheyenne, a workaround is needed to avoid the modules provided by CISL take precedence over the spack modules. The default module path for compilers is removed, the module path is set to a different location and that location is then loaded into the module environment. If new compilers or MPI libraries are
-added to ``/glade/u/apps/ch/modulefiles/default/compilers`` by CISL, the spack-stack maintainers need to make the corresponding changes in ``/glade/work/jedipara/cheyenne/spack-stack/modulefiles/compilers``. See :numref:`Section %s <Preconfigured_Sites_Cheyenne>` for details.
+added to ``/glade/u/apps/ch/modulefiles/default/compilers`` by CISL, the spack-stack maintainers need to make the corresponding changes in ``/glade/work/jedipara/cheyenne/spack-stack/modulefiles/compilers``. See :numref:`Section %s <Preconfigured_Sites_Cheyenne>` for details. Note also that there are problems with newer versions of the Intel compiler/MPI library when trying to run MPI jobs with just one task (``mpiexec -np 1``) - for JEDI, job hangs forever in a particular MPI communication call in oops. This is why an older version Intel 19 is used here and on Casper.
```
Comment (ulmononian) [inline doc/source/MaintainersSection.rst:373]: i'm wondering if we should include an intel 2022 version for use with the ufs and its component apps. not sure  if they will want to revert to intel 19. thoughts? (and apologies for not mentioning this sooner).

**Comment by climbfuji** [comment]
> > 
> 
> I finished installing on all three ParallelWorks sites yesterday. The installs required statically built versions of MAPL and ESMF, but work with the cpld_control_p8 regression test of the WM. I am pretty sure we will need the statically built ESMF/MAPL combination on all the platforms to get the WM to work correctly.

Excellent! Thanks for this, @mark-a-potts . Do you mind updating https://github.com/NOAA-EMC/spack-stack/issues/503 to keep track of the progress? 

I don't see a problem with only static ESMF and MAPL. My suggestion: Let's make the change on the release branches for rolling out 1.3.1 in a few weeks. In the meantime, we can manually modify the installations on the machines that we need to use for ufswm testing with 1.3.0.

**Comment by climbfuji** [comment]
I would say no, but can you ask them? This machine is going away in a few months, and the sysadmins clearly told us that newer Intel versions have been problematic.

File: doc/source/PreConfiguredSites.rst [hunk 2]
```diff
@@ -14,37 +14,37 @@ Officially supported spack-stack installations (tier 1)
...
+| NCAR-Wyoming Casper Intel                                  | Dom Heinzeller / ???          | ``/glade/work/epicufsrt/contrib/spack-stack/spack-stack-1.3.0-casper/envs/unified-env``                      |
++------------------------------------------------------------+-------------------------------+--------------------------------------------------------------------------------------------------------------+
+| NCAR-Wyoming Cheyenne Intel/GNU                            | Cam Book / Dom Heinzeller     | ``/glade/work/epicufsrt/contrib/spack-stack/spack-stack-1.3.0/envs/unified-env``                             |
++------------------------------------------------------------+-------------------------------+--------------------------------------------------------------------------------------------------------------+
+| NOAA Parallel Works (AWS, Azure, Gcloud) Intel/GNU?        | Cam Book / ???                | **missing, will be supported**                                                                               |
```
Comment (ulmononian) [inline doc/source/PreConfiguredSites.rst]: this should be mark as a primary. you can add me as the back-up (unless you want the second maintainer to be from another agency/institution).

Comment (climbfuji) [reply doc/source/PreConfiguredSites.rst]: Be my guest :-) Thanks for doing this.

**Review by srherbener — APPROVED** [review]
Thanks for this update!

File: doc/source/PreConfiguredSites.rst (hunk 2 above)
Comment (climbfuji) [reply doc/source/PreConfiguredSites.rst]: Updated!

**Review by ulmononian — APPROVED** [review]
thanks for these updates, @climbfuji.

**Pull request merged**